src/features/MentorPage/MentorChips/MentorChips.tsx

```diff
@@ -0,0 +1,56 @@
+import { ChipProps } from '../../../components/Chip/types';
+import styled from 'styled-components';
+import Chip from '../../../components/Chip';
+import ShowMoreChips from './ShowMoreChips';
+
+/**
+ * Logic to show skill chips. Array of SkillPillItems
+ * should be passed to SkillPill, where styles and layout
+ * will be added. Should call MentorCards when chips are
+ * selected?
+ */
+
+const MentorSkillChips = ({ items }: { items: Array<ChipProps> }) => {
+  return (
+    <ChipContainer>
+      <SkillChips id="skill-chips">
+        {items.map(item => (
+          // eslint-disable-next-line react/jsx-key
```

Comment:
Is this necessary? Why? I dont get a warning about a key when I run on development without this line